apps/cloud-player/README.md

```diff
@@ -0,0 +1,53 @@
+# Cloud Player
+
+A generic player which autonomously manges audio focuses and player resources.
+
+## URLs
+
+### yoda-app://cloud-player/play
+
+Queries:
+- `text`: String. Optional speech synthesis text.
+- `url`: String. Optional media url.
+- `transient`: Enum `0`, `1`. Optional flag indicates if the media was expected to be requested with transient focus. Defaults to `1`
+- `sequential`: Enum `0`, `1`.  Optional flag indicates if the speech synthesis and media should be played in sequential order but not simultaneously. Defaults to `0`.
+- `mediaId`: Optional media id tag. Cloud Player does nothing with it.
+
+### yoda-app://cloud-player/pause
+
+### yoda-app://cloud-player/resume
+
+### yoda-app://cloud-player/stop
+
+### yoda-app://cloud-player/play-tts-stream
+
+Queries:
+- `pickupOnEnd`:
+- `pickupDuration`:
+
+## Events
+
+### yodaos.apps.multimedia.playback-status
+
+Message: [0, `status-code`]
+
+### yodaos.apps.cloud-player.tts.status
+
+Message: [`status-code`]
+
+## Methods
+
+### yodaos.apps.cloud-player.get-stream-channel
+
+Response: [`utter-id`]
+
+### yodaos.apps.cloud-player.inspect-players
```

Comment:
The above method names seem not to be valid function name, how about using camel-case style? Which we could represent this in SDK, too.